packages/server/src/common/session/client-session-listener.ts

```diff
@@ -13,6 +13,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
+
```

Comment:
Minor, but this just introduces an empty line, which is not required by the linter afaik.